site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -87,6 +87,26 @@ kubectl create -f gameserver.yaml
 kubectl apply -f gameserver.yaml
 ```
 
+{{< alert title="Note" color="info">}}
+If you changed `main.go` again and want to apply the changes to the new game servers, then you also need 
+to modify the `gamerserver.yaml` file's `imagePullPolicy` to be `Always`, or else the new game servers might
+only pull a cached version of the image, which doesn't have the new changes. See the documentation for
+[ImagePullPolicy](https://kubernetes.io/docs/concepts/containers/images/#updating-images) for more details.
```

Comment:
To shorten this a bit, I would just link `imagePullPolicy` above and anyone curious will know to go look at the hyperlink, which would let you drop this sentence.